Hardware/Hardware/CardReader/StripeCardReader/StripeCardReaderService.swift

```diff
@@ -399,7 +404,11 @@ extension StripeCardReaderService: CardReaderService {
             self.readerLocationProvider?.fetchDefaultLocationID { result in
                 switch result {
                 case .success(let locationId):
-                    return promise(.success(LocalMobileConnectionConfiguration(locationId: locationId)))
+                    return promise(.success(LocalMobileConnectionConfiguration(
+                        locationId: locationId,
+                        merchantDisplayName: nil,
+                        onBehalfOf: nil,
+                        tosAcceptancePermitted: options?.builtInOptions?.termsOfServiceAcceptancePermitted ?? true)))
```

Comment:
Because allowing ToS acceptance is the better default. Preventing it is an active suppression step : up until now, we've implicitly said `true` in all cases (by using the other init).

We only want to suppress it in this PR because it would be surprising for a merchant to suddenly see it 3 seconds after they launched the app if they weren't intending to take a payment – that wouldn't happen for everyone, only people who've used TTP before, but it would still be a weird prompt.